CHANGELOG.md

```diff
@@ -18,6 +20,11 @@ unaffected. (#684)
 `optionsTemplateURL` will display the template in the side navigation menu.
 This may require minor layout/appearance adjustments in the template. (#684)
 
++ To upgrade:
+  ++ If using the "app-title" attribute on frame-page directive, that title will be used as the document title
+  ++ If you want a heading on pages that formerly had the app-header, you'll need to use the "page-title" attribute on frame-page directive (or add your own <h1> tag to their pages wherever/however you want)
+  ++ If you were previously using APP_OPTIONS.optionsTemplateURL to set up app-header options, those options will now appear in the side navigation and may likely need a bit of tweaking if you had a rigid CSS layout.
```

Comment:
Likewise. "you" weren't using `APP_OPTIONS.optionsTemplateURL`. Maybe you have a role in developing or  maintaining an app that uses this.